website/docs/cdktf/concepts/providers.mdx

```diff
@@ -19,7 +19,7 @@ You can install pre-built providers, import providers from the Terraform Registr
 
 You can also use the [`provider add` command](/cdktf/cli-reference/commands#provider-add) to add providers to your CDKTF application. It will automatically try to install a pre-built provider if available and fall back to generating bindings locally if none was found.
 
-### Import Providers
+## Import Providers
```

Comment:
hmm it almost feels like we put this heading in the wrong place! I feel like it should come right above the "You can install pre-built providers....." sentence.

Reply:
This isn't your fault, Ansgar - it looks like it was already like that, but I'd like to make this opportunistic fix :)

Reply:
haha, no worries! I'm open for opportunistic fixes!

Reply:
Okay, got it: https://github.com/hashicorp/terraform-cdk/pull/2299/commits/4f03ceb074c9710a406da5cd90d86c927ce6e5ca
(Is it correct? 😄)

Reply:
yay!! I like it so much better! Let me go through quickly and just read it, but thank you for doing this!!